packages/plugins/EVM/src/state/TransactionFormatter/descriptors/ITO.ts

```diff
@@ -13,8 +13,8 @@ export class ITODescriptor implements TransactionDescriptor {
             chainId: context.chainId,
             account: context.from,
         })
-        const token = await connection?.getFungibleToken(context.parameters?._token_addr ?? '')
-        const amount = formatBalance(context.parameters?._total_tokens, token?.decimals)
+        const token = await connection?.getFungibleToken((context.parameters?._token_addr as string) ?? '')
```

Comment:
Why force type casting?

Reply:
There's boolean type in `context.parameters`.

Reply:
That's because we have mismatched the API. Do you need https://github.com/DimensionDev/Maskbook/pull/6690 to merge?